assets/dev/js/editor/controls/gap.js

```diff
@@ -0,0 +1,22 @@
+let ControlDimensionsView = require( 'elementor-controls/dimensions' ),
+	ControlGapItemView;
+
+ControlGapItemView = ControlDimensionsView.extend( {
+
+	ui() {
+		let ui = ControlDimensionsView.prototype.ui.apply( this, arguments );
+
+		ui.controls = '.elementor-control-gap > input:enabled';
+		ui.link = 'button.elementor-link-gaps';
+
+		return ui;
+	},
+
+	getPossibleDimensions() {
+		return [
+			'row',
+			'column',
+		];
+	},
+} );
```

Comment:
Add blank lines above and below `module.exports = ControlGapItemView;`

Reply:
Done